lib/OpenQA/Scheduler/Model/Jobs.pm

```diff
@@ -178,9 +178,9 @@ sub schedule ($self) {
         try {
             $worker = $schema->resultset('Workers')->find({id => $worker_id});
         }
-        catch {
+        catch ($error) {
             log_debug("Failed to retrieve worker ($worker_id) in the DB, reason: $_");    # uncoverable statement
```

Comment:
```suggestion
            log_debug("Failed to retrieve worker ($worker_id) in the DB, reason: $e");    # uncoverable statement
```

Reply:
And the same for all other instances. Try::Tiny uses `$_`, and you can replace that with the named variable.